src/datasets/bigquery/exact_mau/reference.md

```diff
@@ -0,0 +1,215 @@
+restricts to profiles that report usage in the US on Firefox versions 60 or 61.
+There is implicitly no restriction on any other dimensions.
+Thus, the empty slice - "All" - is also a valid slice and simply places no 
+restrictions on any dimension.
+Note there are some complexities here:
+
+- Firstly, a dimension may be scalar and need to be suitably bucketed (instead of every possible profile age being a unique slice element, maybe we prefer to group users between 12 and 16 months old into a single slice element); likewise we may need to use normalized versions of string fields
+- Secondly, we require that dimensions be non-overlapping, especially for metrics calculated over multiple days of user activity.  In a given day, a profile may be active in multiple countries, but we aggregate that to a single value by taking the most frequent value seen in that day, breaking ties by taking the value that occurs last. In a given month, the assigned country may change from day to day; we use the value from the most recent active day up to the day we're calculating usage for.
+
+# Using the Tables
+
+The various Exact MAU datasets are computed daily from the `*_last_seen`
+tables (see [`clients_last_seen`](/datasets/bigquery/clients_last_seen/reference.html))
+and contain pre-aggregated DAU, WAU, and MAU counts per usage criterion
+per each unique combination of dimensions values.
+Because of our restriction that dimension values be non-overlapping, we
+can recover MAU for a particular slice of the data by summing over all rows
+matching the slice definition.
+
+The simple case of retrieving DAU/WAU/MAU for usage criterion
+"All Desktop Activity" and slice "All" looks like:
+
+```sql
+SELECT
+    submission_date,
+    SUM(mau) AS mau
+FROM
+    `moz-fx-data-derived-datasets.telemetry.firefox_desktop_exact_mau28_by_dimensions_v1`
+GROUP BY
+    submission_date
+ORDER BY
+    submission_date
+```
+
+Now, let's refine our slice to "Country = US; Campaign = whatsnew"
+via a `WHERE` clause:
+
+```sql
+SELECT
+    submission_date,
+    SUM(mau) AS mau
+FROM
+    `moz-fx-data-derived-datasets.telemetry.firefox_desktop_exact_mau28_by_dimensions_v1`
+WHERE
+    country = 'US'
+    AND campaign = 'whatsnew'
+GROUP BY
+    submission_date
+ORDER BY
+    submission_date
+```
+
+Perhaps we want to compare MAU as above to  [aDAU](/cookbooks/active_dau.html)
+over the same slice. The column `visited_5_uri_dau` gives DAU as calculated
+with the "Desktop 5 URI Active" usage criterion, corresponding to aDAU:
+
+```
+SELECT
+    submission_date,
+    SUM(mau) AS mau,
+    SUM(visited_5_uri_dau) AS adau
+FROM
+    `moz-fx-data-derived-datasets.telemetry.firefox_desktop_exact_mau28_by_dimensions_v1`
+WHERE
+    country = 'US'
+    AND campaign = 'whatsnew'
+GROUP BY
+    submission_date
+ORDER BY
+    submission_date
+```
+
+Additional usage criteria may be added in the future as new columns named
+`mau_*`, etc. where the suffix describes the usage criterion.
```

Comment:
right now we are using the prefix to describe the criterion

Reply:
Yes, I intended to change this. Thanks for catching.